bpf/kmesh/workload/include/tail_call.h

```diff
@@ -16,6 +26,14 @@ struct {
     __uint(map_flags, 0);
 } map_of_tail_call_prog SEC(".maps");
 
+struct {
+    __uint(type, BPF_MAP_TYPE_PROG_ARRAY);
+    __uint(key_size, sizeof(__u32));
+    __uint(value_size, sizeof(__u32));
+    __uint(max_entries, MAP_SIZE_OF_TAIL_CALL_PROG);
+    __uint(map_flags, 0);
+} map_of_tail_call_prog_for_xdp SEC(".maps");
```

Comment:
The map name is too long and will be truncated when used bpftool

Reply:
> The map name is too long and will be truncated when used bpftool

change to xdp_tailcall_map